doc/user/pathd.rst

```diff
@@ -6,11 +6,168 @@ PATH
 
 :abbr:`PATH` is a daemon that handles the installation and deletion
 of Segment Routing (SR) Policies.
+Based in MPLS (This means that your OS of choice must support MPLS),
+SR add a stack of MPLS labels to ingress packets so these
+packets are egress through the desired path.
+
+.. image:: images/pathd_general.png
+
+The SR policies and Segment Lists can be configured either locally by means
+of vtysh or centralized based in a SDN controller (ODL, Cisco, ...) and
+communicated using the PCEP protocol (:rfc:`5440`).
```

Comment:
'centralized based through an SDN Controller (ODL, Cisco, ...) communicating using the PCEP protocol ...'